rotkehlchen/chain/evm/decoding/gearbox/gearbox_cache.py

```diff
@@ -277,8 +296,21 @@ def query_gearbox_data_from_chain(
     pools_data = evm_inquirer.contracts.contract(
         CHAIN_ID_TO_DATA_COMPRESSOR[evm_inquirer.chain_id],
     ).call(node_inquirer=evm_inquirer, method_name='getPoolsV3List')
-    new_pools = []
+    new_pools: list[GearboxPoolData] = []
+    last_notified_ts = 0
     for pool_data in pools_data:
+        if (current_time := ts_now()) - last_notified_ts > 5:
```

Comment:
do you really need to send it every 5 seconds? Can't it be every 10-15 processed entries? Asking for the tiemstamp is slower than checking the %

Reply:
I kept it timewise because it's simple this way to avoid flooding the WS messages. and it ensures that user see the progress not too faster/slower than required